core/src/main/resources/hudson/model/Messages_fr.properties

```diff
@@ -157,28 +342,69 @@ ChoiceParameterDefinition.MissingChoices=Choix requis.
 RunParameterDefinition.DisplayName=Paramètre d''exécution
 PasswordParameterDefinition.DisplayName=Paramètre "Mot de passe"
 
-Node.Mode.NORMAL=Utiliser ce noeud autant que possible
-Node.Mode.EXCLUSIVE=Réserver ce noeud uniquement pour les jobs qui lui sont attachés
+Node.BecauseNodeIsReserved=‘{0}’ est réservé pour les jobs avec une expression de libellé correspondant
+Node.BecauseNodeIsNotAcceptingTasks=‘{0}’ n''accepte pas de tâche
+Node.LabelMissing=‘{0}’ n''a pas le libellé ‘{1}’
+Node.LackingBuildPermission=‘{0}’ ne possède pas le droit de s''exécuter sur ‘{1}’
+Node.Mode.NORMAL=Utiliser ce nœud autant que possible
+Node.Mode.EXCLUSIVE=Réserver ce nœud uniquement pour les jobs qui lui sont attachés
 
 ListView.DisplayName=Vue liste
 
 MyView.DisplayName=Ma vue
 
-LoadStatistics.Legends.TotalExecutors=Nombre total d''exécuteurs
+LoadStatistics.Legends.DefinedExecutors=Éxecuteurs déclarés
 LoadStatistics.Legends.OnlineExecutors=Nombre d''exécuteurs en ligne
+LoadStatistics.Legends.ConnectingExecutors=Nombre d''éxecuteurs en cours de connexion
+LoadStatistics.Legends.TotalExecutors=Nombre total d''exécuteurs
 LoadStatistics.Legends.BusyExecutors=Nombre d''exécuteurs occupés
+LoadStatistics.Legends.IdleExecutors=Nombre d''exécuteurs en attente
 LoadStatistics.Legends.AvailableExecutors=Nombre d''exécuteurs disponibles
 LoadStatistics.Legends.QueueLength=Longueur de la file d''attente
 
 Cause.LegacyCodeCause.ShortDescription=Ce job a été lancé par du code legacy. Pas d''information sur les causes
 Cause.UpstreamCause.ShortDescription=Démarré par le projet amont "{0}" de numéro de build {1}
+Cause.UpstreamCause.CausedBy=originally caused by:
 Cause.UserCause.ShortDescription=Démarré par l''utilisateur {0}
 Cause.UserIdCause.ShortDescription=Démarré par l''utilisateur {0}
-Cause.RemoteCause.ShortDescription=Démarré à distance par {0}
+Cause.RemoteCause.ShortDescription=Démarré à distance par l''hôte {0}
+Cause.RemoteCause.ShortDescriptionWithNote=Demarré à distance par l''hôte {0} avec le commentaire : {1}
+
+ProxyView.NoSuchViewExists=La vue globale {0} n''existe pas
+ProxyView.DisplayName=Inclure une vue globale
 
 MyViewsProperty.DisplayName=Mes vues
 MyViewsProperty.GlobalAction.DisplayName=Mes vues
+MyViewsProperty.ViewExistsCheck.NotExist=Inexistance de la vue avec le nom {0}
+MyViewsProperty.ViewExistsCheck.AlreadyExists=La vue avec le nom {0} existe déjà
 
-ManageJenkinsAction.DisplayName=Administrer Jenkins
-ParametersDefinitionProperty.DisplayName=Ce build a des paramètres
+CLI.restart.shortDescription=Redémarrer Jenkins
+CLI.safe-restart.shortDescription=Redémarrer Jenkins en sûreté
+CLI.keep-build.shortDescription=Conserver cette construction sans limite de temps.
+
+BuildAuthorizationToken.InvalidTokenProvided=Le token fourni est invalide.
+
+Jenkins.CheckDisplayName.NameNotUniqueWarning=Le nom affiché, "{0}", est utilisé comme nom de job et peut causer des résultats de recherche erronés.
```

Comment:
```suggestion
Jenkins.CheckDisplayName.NameNotUniqueWarning=Le nom affiché "{0}" est utilisé comme nom de job et peut causer des résultats de recherche erronés.
```

Reply:
Same here: https://github.com/jgreffe/jenkins/blob/2fc718da2a377d9dc411ebae95badf2f42212bef/core/src/main/resources/hudson/model/Messages.properties#L402

Reply:
Same as above, I would remove them anyway for the french translation.